### PR TITLE
US234004 pfe-nav noscript styles & browser fixes

### PR DESCRIPTION
**Pull request by chrisdo1**
* Added pfe-navigation--noscript.scss file

## PFE-nav noscript styles

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Style bug fixes

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. https://docs.google.com/document/d/1HTIknZeZeeM1d6pIyyvpQ7iKzBWNE8knvGlvl3zZT3s/edit#

#### Browser requirements
Your component should work in all of the following environments:

- [ ] Latest 2 versions of Edge
- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
- [ ] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [ ] Latest 2 versions of Safari
- [ ] Galaxy S9 Firefox
- [ ] iPhone X Safari
- [ ] iPad Pro Safari
- [ ] Pixel 3 Chrome

Your repository infrastructure updates should work for at least:
- [ ] Node v8.x
- [ ] NPM v7.x

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Did browser testing pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?
- [ ] Did you update the CHANGELOG.md file with a summary of this update?

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 1]
```diff
@@ -38,16 +44,85 @@
...
     }
-    .pfe-navigation-item__tray--container {
-        margin: 0 auto;
+
+    [slot="trigger"] {
+        margin: 0;
+    }
+
+    // Override grid so that CTAs do not overlap.
+    .pfe-navigation--footer {
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]: @chrisdo1 can you change all the `.pfe-navigation--footer` classes to `.pfe-navigation--footer:not(:empty)` so the styles only apply when CTAs are present? Thanks!

Comment (chrisdo1) [reply elements/pfe-navigation/src/pfe-navigation--lightdom.scss]: Fixed. Thanks.

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 2]
```diff
@@ -38,33 +44,97 @@
...
+
+    [is_nested][parent_hidden] .pfe-navigation-item__tray {
+        max-height: none;
+        overflow: visible;
+    }
+
+    @media screen and (max-width: 1023px) {
+        .pfe-navigation__main {
+            display: block !important;
+        }
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]:
```suggestion
        ul {
            align-items: center;
        }
```

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 3]
```diff
@@ -2,31 +2,37 @@
...
         [slot="trigger"] {
-            font-size: 12px;
+            font-size: 12px; 
+            margin: 0;
 
             a {
                 color: #{pfe-color(surface--darkest--text)};
             }
         }
+        // Accordion trigger styles when expanded.
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]:
```suggestion
        // Accordion trigger styles when expanded.
        > * {
            margin: 0 10px 0 0;
        }
```

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss [hunk 4]
```diff
@@ -38,33 +44,97 @@
...
+            margin: 0 auto;
+        }
+    }
+
+    [is_nested][parent_hidden] .pfe-navigation-item__tray {
+        max-height: none;
+        overflow: visible;
+    }
+
+    @media screen and (max-width: 1023px) {
```
Comment (starryeyez024) [inline elements/pfe-navigation/src/pfe-navigation--lightdom.scss]: @chrisdo1 can we bump up this breakpoint a bit, the links are crowded until like 1100px or so
https://web-dev-drupal-webux2.dev.a1.vary.redhat.com/es/selecciona-un-idioma

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss (hunk 3 above)
Comment (starryeyez024) [reply elements/pfe-navigation/src/pfe-navigation--lightdom.scss]: I'm hoping these dont impact regular styles when javascript is enabled. If they do, forget about it :)

File: elements/pfe-navigation/demo/index.html [hunk 5]
```diff
@@ -36,7 +38,7 @@
 
   <noscript>
     <link href="../../pfelement/pfelement--noscript.min.css" rel="stylesheet">
-    <link href="../pfe-navigation--noscript.css" rel="stylesheet">
+    <link href="../dist/pfe-navigation--noscript.css" rel="stylesheet">
```
Comment (castastrophe) [inline elements/pfe-navigation/demo/index.html]: Can you make this same update to the storybook page for navigation too?

File: elements/pfe-navigation/src/pfe-navigation.js [hunk 6]
```diff
@@ -160,14 +160,11 @@ class PfeNavigation extends PFElement {
     if (state) {
       // Add the overlay to the page
       this._overlay.removeAttribute("hidden");
-      // This prevents background scroll while nav is open
-      document.body.style.overflow = "hidden";
```
Comment (castastrophe) [inline elements/pfe-navigation/src/pfe-navigation.js]: This doesn't look quite right to me - is this the latest?  Without the overflow hidden on the body, doesn't the scroll move behind the overlay?

File: elements/pfe-navigation/src/pfe-navigation.js [hunk 7]
```diff
@@ -201,6 +199,9 @@ class PfeNavigation extends PFElement {
   connectedCallback() {
     super.connectedCallback();
 
+    // Store the width of the scrollbar, if it exists
+    this.scrollbarWidth = window.innerWidth - document.documentElement.clientWidth;
```
Comment (castastrophe) [inline elements/pfe-navigation/src/pfe-navigation.js]: This isn't being used in this code - do we still need it?

File: elements/pfe-navigation/src/pfe-navigation.js [hunk 8]
```diff
@@ -270,10 +271,11 @@ class PfeNavigation extends PFElement {
 
   _stickyHandler() {
     if(window.pageYOffset >= this.top) {
-      this.classList.add("sticky");
+      this.classList.add("pfe-sticky");
     } else {
-      this.classList.remove("sticky");
+      this.classList.remove("pfe-sticky");
     }
+    // this.stuck = window.pageYOffset >= this.top;
```
Comment (castastrophe) [inline elements/pfe-navigation/src/pfe-navigation.js]: Remove this if the getter and setter are gone

File: elements/pfe-navigation/src/pfe-navigation--lightdom.scss (hunk 1 above)
Comment (starryeyez024) [reply elements/pfe-navigation/src/pfe-navigation--lightdom.scss]: +1 tested

**Pull request merged**